### PR TITLE
Fix for multi registration bug

### LINKED ISSUE #241: Is something wrong with mainnet1.ddns.tabookey.com?
I was looking over the blockchain to see if my node is behaving and perhaps getting some traffic. 
 I noticed there was a flurry of activity some hours ago on the contract all from mainnet1.ddns.tabookey.com 

In the space of a few minutes it created the following transactions all of which were calls to register relay

https://etherscan.io/tx/0x974b13fcb9dcc7e27b2643f1a92cd8c4f68f5870b726ab56ab24b382b3851688
https://etherscan.io/tx/0xf0511dd58646714ae5798d54722b71df1d915579515e466bbb0988d5843802b7
https://etherscan.io/tx/0x43e4f29f87a12140159518a16977c90b232cd24f4e495e9080791b9128ee4eca
https://etherscan.io/tx/0x08181eb8723627aaa85871022202f0d8e203b1ec09cfe78fe6f9c216fa4f685e
https://etherscan.io/tx/0xf39fb63c2dc3d7ceb869edccb1dc6f18e1d552f8f0c73d40f289bb882e5cdf0e
https://etherscan.io/tx/0x1e5c0c51813940f99be125d88ac4a1561a4690083fdd04c118ab50b7efbb5222
https://etherscan.io/tx/0xff73e9468a8de366dd4cf5fc8834f65c0af4672e476e71587093d8787d56853c

Did something go wrong with the node or is this a bug?

### PR DESCRIPTION
**Pull request by sshelton76**
Fixes #241 by implementing the recommended changes to the keepAlive method

### CODE REVIEW

File: server/src/relay/RelayHttpServer.go [hunk 1]
```diff
@@ -201,7 +204,7 @@ func parseCommandLine() (relayParams librelay.RelayParams) {
 	relayParams.DefaultGasPrice = *defaultGasPrice
 	relayParams.GasPricePercent = big.NewInt(*gasPricePercent)
 	relayParams.RegistrationBlockRate = *RegistrationBlockRate
-	relayParams.EthereumNodeURL = *ethereumNodeUrl
+	relayParams.ethereumNodeURL = *ethereumNodeURL
```
Comment (shahafn) [inline server/src/relay/RelayHttpServer.go]: Not sure this refactoring should be in this PR, or even how it compiles as relayParams has no such member.

File: server/src/relay/RelayHttpServer.go [hunk 2]
```diff
@@ -4,25 +4,28 @@ import (
...
-	"librelay/txstore"
 	"log"
 	"math/big"
 	"net/http"
 	"net/url"
 	"os"
 	"path/filepath"
 	"time"
+
+	"../librelay"
```
Comment (shahafn) [inline server/src/relay/RelayHttpServer.go]: Why do the import paths need to change here? Are you sure that you defined your GOPATH correctly?

File: server/src/relay/RelayHttpServer.go [hunk 3]
```diff
@@ -177,8 +180,8 @@ func parseCommandLine() (relayParams librelay.RelayParams) {
 	relayHubAddress := flag.String("RelayHubAddress", "0xD216153c06E857cD7f72665E0aF1d7D82172F494", "RelayHub address")
 	defaultGasPrice := flag.Int64("DefaultGasPrice", int64(params.GWei), "Relay's default gasPrice per (non-relayed) transaction in wei")
 	gasPricePercent := flag.Int64("GasPricePercent", 10, "Relay's gas price increase as percentage from current average. GasPrice = (100+GasPricePercent)/100 * eth_gasPrice() ")
-	RegistrationBlockRate:= flag.Uint64("RegistrationBlockRate", 6000-200, "Relay registeration rate (in blocks)")
-	ethereumNodeUrl := flag.String("EthereumNodeUrl", "http://localhost:8545", "The relay's ethereum node")
+	RegistrationBlockRate := flag.Uint64("RegistrationBlockRate", 6000-200, "Relay registeration rate (in blocks)")
+	ethereumNodeURL := flag.String("ethereumNodeURL", "http://localhost:8545", "The relay's ethereum node")
```
Comment (drortirosh) [inline server/src/relay/RelayHttpServer.go]: Note that this "refactoring" breaks the configuration API, as it changes the command-line parameter. please remove all those refactorings from the PR.

Comment (sshelton76) [reply server/src/relay/RelayHttpServer.go]: Woah!  None of that was in this PR originally, looks like somehow my linting branch got merged in with the very, very simple fix I actually pushed in the PR.  The PR must be tracking my master branch even after the PR is submitted.  You would tend to think a PR would be tied to the HEAD as of the time of the submission.

I'll back this out and see if it tracks it again, if not I'll rebase my master branch and wait for the PR to be completed.  Sorry!

**Comment by sshelton76** [comment]
FYI all this junk you're seeing is an effort to try and get your code to pass golint.  :)

This particular PR just got munged somehow with my linting branch.

**Comment by sshelton76** [comment]
Let's reject this PR, I'll rebase, repatch, resubmit and wait.

**Comment by shahafn** [comment]
looks good, can you please increment the minor version number on `RelayHttpServer.go`?

**Comment by drortirosh** [comment]
@sshelton76, can you increment the version number, as per Shahaf's request, so we can merge it in?

**Pull request merged**